lesson2/README.md

```diff
@@ -27,14 +27,28 @@ When you interact with an app on your phone, there are different "screens" that
 
 #### [Activities](https://developer.android.com/guide/components/activities.html) (ex. MainActivity.java)
 
-Activities contain one or many fragments inside of them.  It handles the passing of information between fragments.  Most simple apps (like the ones you'll be making in the beginning of this class) will only need one activity.
+ An activity provides a screen that users can interact with in order to do something, such as dial the phone, take a photo, send an email, or view a map. Each activity is given a window in which to draw its UI. The window typically fills the screen, but may be smaller than the screen and float on top of other windows.
+ 
+ Each activity can then start another activity in order to perform different actions. You generally have the main activity which launches other activities (or, your application could just be a single activity). Each time a new activity starts, the previous activity is stopped, but the system preserves the activity in a stack (the "back stack"). When a new activity starts, it is pushed onto the back stack and takes user focus.
+ 
+ Note that an activity can hold multiple "fragments," which is very useful. The activity handles passing information between fragments and lets you switch between different fragments.
```

Comment:
i think only the first paragraph is a somewhat abbreviated version of the websites. Since this is the most accurate description I thought it'd fit, but rewording things is fine if that's what you're asking